index.js

```diff
@@ -47,6 +47,19 @@ File.prototype.isDirectory = function() {
   return this.isNull() && this.stat && this.stat.isDirectory();
 };
 
+File.prototype.toJSON = function() {
+  var self = this;
+  return {
+    basename: self.basename,
+    contents: self.contents.toString(),
+    dirname: self.dirname,
+    extname: self.extname,
+    path: self.path,
+    relative: self.relative,
+    stem: self.stem,
```

Comment:
I'm not sure.

I never use trailing commas like that, but it did appear to conform to the project style.

such as: https://github.com/gulpjs/vinyl/blob/master/index.js#L80